ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/priv/PrivGetLogs.java

```diff
@@ -70,13 +71,27 @@ public JsonRpcResponse response(final JsonRpcRequestContext requestContext) {
         filter
             .getBlockHash()
             .map(
-                blockHash ->
-                    privacyQueries.matchingLogs(privacyGroupId, blockHash, filter.getLogsQuery()))
+                blockHash -> {
```

Comment:
We could encapsulate the `map` and `orElseGet` blocks into their own functions to make this main function a bit cleaner. Something like `findLogsForBlockHash` and `findLogsForBlockRange`

Reply:
done